### DISCUSSION TITLE
Stop and Continue response like llmstudio

### DISCUSSION THREAD

**Discussion by electro199** [ggml-org/llama.cpp#8969] [Q&A]
In LLM studio there is option to stop generation edit the response and then continue so LLM use the provided text/token for completion. How to replicate that in the llama-server ?

**Comment by bobleer** [comment]
the same question.
how to stop a request in llama-server?

**Comment by ExtReMLapin** [comment]
In llama-server, to stop a request, just close the connection.

To continue it, just resent the whole prompt (so it doesn't work with chat endpoint, eh) with the partially generated message you saved
